introduction.md

```diff
@@ -0,0 +1 @@
+hello its me new student in codeschool mastering github :D
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.